android/proguard-rules.pro

```diff
@@ -24,7 +24,10 @@
 
 # Keep the classes/members we need for client functionality.
 -keep @interface app.notifee.core.KeepForSdk
--keep @app.notifee.core.KeepForSdk class *
+-keep @app.notifee.core.KeepForSdk class * {
+  public *;
+  protected *;
+}
```

Comment:
I did a quick test after removing this change. It still works. In my view, you can remove this change. Apologies for the irrelevant change.

Reply:
Great, thanks for checking that. Removes testing requirements to avoid changes here, so I've locally reverted this change and will push it with the rest of my local changes in a moment